libnetdata/buffer/buffer.c

```diff
@@ -160,6 +160,31 @@ void buffer_strcat(BUFFER *wb, const char *txt)
     }
 }
 
+void buffer_strcat_jsonescape(BUFFER *wb, const char *txt)
```

Comment:
You need to put `void buffer_strcat_jsonescape(BUFFER *wb, const char *txt)` in a header file to avoid `implicit decleration` errors.

Reply:
Ah. Did not see those. Will add...